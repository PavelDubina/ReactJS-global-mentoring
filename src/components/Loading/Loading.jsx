import React from 'react'
import styles from '../../../styles/Loading.module.scss'

export const Loading = () => (
  <div className={styles.lds_spinner}>
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
)
