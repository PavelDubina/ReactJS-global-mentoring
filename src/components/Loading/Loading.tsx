import React from 'react'
import styles from '../../../styles/Loading.module.scss'

export const Loading: React.FC = () => (
  <div className={styles.lds_spinner}>
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
)
