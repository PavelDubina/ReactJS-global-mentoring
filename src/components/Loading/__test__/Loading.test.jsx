import React from 'react'
import { create } from 'react-test-renderer'
import { Loading } from '../Loading'

describe('Loading', () => {
  it('Should renders correctly', () => {
    const component = create(<Loading />).toJSON()
    expect(component).toMatchSnapshot()
  })
})
