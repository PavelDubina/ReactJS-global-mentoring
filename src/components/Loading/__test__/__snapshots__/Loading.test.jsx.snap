// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Should renders correctly 1`] = `
<div
  className="lds_spinner"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
